src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer extends Bulldozer implements MachineProducer {
+
+    @Override
+    public List<? super Bulldozer> get() {
+        List<Bulldozer> bulldozer = new ArrayList<>();
+        bulldozer.add(new Bulldozer());
+        bulldozer.add(new Bulldozer(10,12));
+        return bulldozer;
```

Comment:
not a mistake, I guess, but you dont have to create new ArrayList here
```suggestion
        return List.of(new Bulldozer(...), new Bulldozer());
```

Reply:
I think it's the same